### ISSUE TITLE
Multiple modals with different width

### ISSUE THREAD

**Issue by shubh0602** [powmedia/backbone.bootstrap-modal#55]
Your plugin has proved to be great help. Thanks.

Can I know, if I have three modals with different width how can I handle it? Is there anything which is configurable?

Its not an issue, actually. I may be hitting wrong place, please let me know if this is true.
Regards,

**Comment by shubh0602** [comment]
What I have done now is , I have created an extra property 'customCss' which will hold CSS class
 `initialize: function (options) {
            this.options = _.extend({
                title: null,
                okText: 'OK',
                focusOk: true,
                okCloses: true,
                cancelText: 'Cancel',
                showFooter: true,
                allowCancel: true,
                escape: true,
                animate: false,
                template: template,
                enterTriggersOk: false,
                customCss: ''
            }, options);
        },`
`var template = _.template('\
    <div class="modal-dialog {{customCss}}"><div class="modal-content">\`

Hope this is correct. Any suggestions?

**Comment by powmedia** [comment]
Yes that could work; you could also provide a custom template with the `template` option.  But for your case a CSS class is probably best
